packages/types/src/lib/interfaces.ts

```diff
@@ -1258,6 +1250,18 @@ export interface PKPBaseProp {
   litActionIPFS?: string;
   litActionJsParams?: any;
   provider?: Provider;
+  controllerSessionSigs?: SessionSigs;
+
+  // -- soon to be deprecated
+  /**
+   * @deprecated - use authContext
+   */
+  controllerAuthMethods?: AuthMethod[];
```

Comment:
This is used for pkpSessionSigs (also litActionSessionSigs)?

Reply:
no - regardless of what sessionSigs they are passed to `controllerSessionSigs` param

Reply:
But how do you get the sessionSig for PKP? Do you get the PKP sessionSig first and pass that? I though that was happening inside this function and the user simply passes their authMethods here?

Reply:
yea, using the usual flow of `getSessionSigs` or `getPkpSessionSigs` and pass that to the pkpEthers function. The `authContext` as I mentioned above needs some fundamental improvement, that's why is not promoted in the doc.